data/pchome

```diff
@@ -0,0 +1,4 @@
+ecimg.tw
+full:ecapi-pchome.cdn.hinet.net
+pchome.com.tw
+pcloud.tw
```

Comment:
I cannot resolve `pcloud.tw` on my Hong Kong VPS.
Is that correct?

Reply:
try `apid.pcloud.tw` instead. However, both `pcloud.tw` and `apid.pcloud.tw` are resolve-able with my local DNS and HK VPS.